src/main/resources/cancellationTasks_IA_Asylum.dmn

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<definitions xmlns="https://www.omg.org/spec/DMN/20191111/MODEL/" xmlns:dmndi="https://www.omg.org/spec/DMN/20191111/DMNDI/" xmlns:dc="http://www.omg.org/spec/DMN/20180521/DC/" id="Definitions_0zkerea" name="DRD" namespace="http://camunda.org/schema/1.0/dmn" exporter="Camunda Modeler" exporterVersion="4.2.0">
+  <decision id="CancellationTask_IA_Asylum" name="Cancellation process DMN">
+    <decisionTable id="DecisionTable_0z3jx1g" hitPolicy="COLLECT">
+      <input id="Input_1" label="From State">
+        <inputExpression id="InputExpression_1" typeRef="string">
+          <text>fromState</text>
+        </inputExpression>
+      </input>
+      <input id="InputClause_0ain215" label="Event">
+        <inputExpression id="LiteralExpression_0h2l2zj" typeRef="string">
+          <text>event</text>
+        </inputExpression>
+      </input>
+      <input id="InputClause_0bomy63" label="State">
+        <inputExpression id="LiteralExpression_1g5glfi" typeRef="string">
+          <text>state</text>
+        </inputExpression>
+      </input>
+      <output id="OutputClause_18zgoaj" label="Action" name="action" typeRef="string" />
+      <output id="OutputClause_0dqzted" label="Task Categories" name="TaskCategories" typeRef="string" />
+      <rule id="DecisionRule_0w82l1w">
+        <inputEntry id="UnaryTests_1nfllzy">
+          <text></text>
+        </inputEntry>
+        <inputEntry id="UnaryTests_0a7tgdv">
+          <text>"EndTheAppeal","MoveTheAppealOffline"</text>
```

Comment:
Can you check these are the correct eventIds used in IAC please?  Check the ccd definition.  I think Gareth has specified the `name` rather than the `id` which is what we should use.  I suspect it is something like `endAppeal`.